src/fabric/src/fabric2_txids.erl

```diff
@@ -136,7 +137,8 @@ clean(St, NeedsSweep) ->
 
 sweep(Tx, {Mega, Secs, Micro}) ->
     Root = erlfdb_directory:root(),
-    CouchDB = erlfdb_directory:create_or_open(Tx, Root, [<<"couchdb">>]),
+    Dir = fabric2_server:fdb_directory(),
```

Comment:
We have these two lines in a lot of places. Wouldn't it make sense to just convert it to a single call from fabric_server:
`fabric2_server:create_or_open_with_dir(Tx, Root)`

Reply:
Yeah definitely. Or even have something like `fabric2_server:get_fdb_prefix(Tx)` as we mostly just want the `Prefix` part